src/containers/DevSite.jsx

```diff
@@ -181,6 +187,7 @@ class Devsite extends Component {
         <div className='nav-shaper'>
           <Nav />
         </div>
+        {this.state.hasCreds ? null : <EnvKeyBanner />}
```

Comment:
Just do 
```
this.state.hasCreds && (
  <EnvKeyBanner />
)
```

Reply:
```
!this.state.hasCreds && (
  <EnvKeyBanner />
)
```
had to inverse the state since `hasCreds` is false on default now and I want to show the banner if there is no .env configs. but thank you will be using this.